### PR TITLE
PMM-8307 upgrade nginx

### PR DESCRIPTION
**Pull request by nikita-b**
https://github.com/Percona-Lab/pmm-submodules/pull/1872

### CODE REVIEW

File: rhel/SPECS/pmm-server.spec [hunk 1]
```diff
@@ -63,13 +61,14 @@ rm -rf %{pmm_repo}-%{pmm_commit}
...
 %{_sysconfdir}/supervisord.d
 %{_sysconfdir}/alertmanager.yml
-%{_sysconfdir}/nginx/conf.d/pmm.conf
-%{_sysconfdir}/nginx/conf.d/pmm-ssl.conf
 %{_datadir}/percona-dashboards/import-dashboards.py*
 %{_datadir}/%{name}
 
 
 %changelog
+* Tue Jun 30 2021 Nikita Beletskii <nikita.beletskii@percona.com> - 2.20.0-2
```
Comment (atymchuk) [inline rhel/SPECS/pmm-server.spec:69]: This date would lead to build error

File: rhel/SPECS/pmm-server.spec [hunk 2]
```diff
@@ -39,8 +39,6 @@ make build-installation-wizard
 %install
 tar -zxvf %SOURCE1
 install -d %{buildroot}%{_sysconfdir}/nginx/conf.d
-mv nginx.conf %{buildroot}%{_sysconfdir}/nginx/conf.d/pmm.conf
```
Comment (atymchuk) [inline rhel/SPECS/pmm-server.spec:42]: Do we need the line above?

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: rhel/SPECS/pmm-server.spec (hunk 1 above)
Comment (nikita-b) [reply rhel/SPECS/pmm-server.spec:69]: Do you mean the day of the week? It just has format check, but correctness but it's a mistake anyway. Fixed!

Comment (atymchuk) [reply rhel/SPECS/pmm-server.spec]: I mean, in the past the builds used to fail, since the builder checks the dates for correctness.

**Review by atymchuk — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**